spec/koans.js

```diff
@@ -1,594 +1,583 @@
+      obj.x = 2;
+      expect(obj.x).toBe(2);
     });
-    
   });
-
 });
 
-describe('a template string, is wrapped in ` (backticks) instead of \' or ". ', () => {
-
-  describe('by default, behaves like a normal string', function() { 
-    it('just surrounded by backticks', function() {
+describe("a template string, is wrapped in ` (backticks) instead of ' or \". ", () => {
+  describe("by default, behaves like a normal string", function() {
+    it("just surrounded by backticks", function() {
       /*let str = ??????*/
-      //expect(str).toEqual('like a string');
+      let str = `like a string`;
+      expect(str).toEqual("like a string");
     });
-    
   });
 
   let x = 42;
   let y = 23;
-  
+
   describe('can evaluate variables, which are wrapped in "${" and "}"', function() {
-    
     it('e.g. a simple variable "${x}" just gets evaluated', function() {
-      let evaluated = `x=x`
-      //expect(evaluated).toBe('x=' + x);
+      let evaluated = `x=${x}`;
+      expect(evaluated).toBe("x=" + x);
     });
-    
-    it('multiple variables get evaluated too', function() {
-      var evaluated = `x+y`;
-      //expect(evaluated).toBe(x + '+' + y);
+
+    it("multiple variables get evaluated too", function() {
+      var evaluated = `${x}+${y}`;
+      expect(evaluated).toBe(x + "+" + y);
     });
-    
   });
 
   describe('can evaluate any expression, wrapped inside "${...}"', function() {
-    
     it('all inside "${...}" gets evaluated', function() {
-      var evaluated = Number(`x+y`);
-      //expect(evaluated).toBe(x+y);
+      var evaluated = Number(`${x + y}`);
+      expect(evaluated).toBe(x + y);
     });
-    
+
     it('inside "${...}" can also be a function call', function() {
-      function getSchool(){ 
-        return 'Ironhack'; 
+      function getSchool() {
+        return "Ironhack";
       }
-      var evaluated = `getSchool()`;
-      //expect(evaluated).toBe('Ironhack');
+      var evaluated = `${getSchool()}`;
+      expect(evaluated).toBe("Ironhack");
     });
-    
   });
-  
 });
 
-describe('arrow functions. ', () => {
-
-  it('are shorter to write', function() {
+describe("arrow functions. ", () => {
+  it("are shorter to write", function() {
     let func = () => {
       /*........*/
+      return "I am func";
     };
-    //expect(func()).toBe('I am func');
+    expect(func()).toBe("I am func");
   });
 
-  it('a single expression, without curly braces returns too', function() {
+  it("a single expression, without curly braces returns too", function() {
     /*let func = () => .........;*/
-    //expect(func()).toBe('I return too');
+    let func = () => "I return too";
+    expect(func()).toBe("I return too");
   });
 
-  it('one parameter can be written without parens', () => {
-   /* let func = ........;*/
-    //expect(func(25)).toEqual(24)
+  it("one parameter can be written without parens", () => {
+    /* let func = ........;*/
+    //let func = num => num = num--;  WHY IS NOT WORKING!!
```

Comment:
For you to return 24 from this operation you need to invert the operator like this: ```--num``` See more info here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Increment